rgw/v2/tests/s3_swift/configs/test_bucket_notification_kafka_none_version_copy_del.yaml

```diff
@@ -0,0 +1,25 @@
+# version enabled + none type + persistent = false - CEPH-83574074
+config:
+ user_count: 1
+ bucket_count: 2
+ objects_count: 2
+ version_count: 5
```

Comment:
@mkasturi18 , we are not looping through version count while creating objects in code, could you please add that code. if that is not required could you please increase objects_count in all configs

Reply:
@hmaheswa  thank you. I have added to loop on version_count. 
Pass logs: http://magna002.ceph.redhat.com/ceph-qe-logs/madhavi/PRs/notification/versioned_bkt/new_logs/